app/validators/api/v3/prescription_drug_payload_validator.rb

```diff
@@ -8,6 +8,9 @@ class Api::V3::PrescriptionDrugPayloadValidator < Api::V3::PayloadValidator
     :is_protocol_drug,
     :patient_id,
     :facility_id,
+    :frequency,
+    :duration_in_days,
+    :teleconsultation_id,
```

Comment:
I suppose we don't need to explicitly omit `teleconsultation_id` from the response payload for the older apps? Shall we confirm?

Reply:
That did get me thinking, though. Should we authorize setting `teleconsultation_id` to just telemedicine MOs?

Reply:
Makes sense. A nurse cannot see the teleconsult prescription screen, so it should not be possible for them to add `teleconsultation_ids` but we could add a check nonetheless. There's another case worth considering:
- A user who had `can_teleconsult` true logs a teleconsult with some drugs
- User's `can_teleconsult` permission is revoked from server side
- User tries to sync the drugs from the teleconsult after and fails

Reply:
I think that's ok, and the intended outcome, right?

Reply:
In this case we should allow the user to log those drugs, I think. The drugs _were_ prescribed to the patient and maybe it's better to collect that information than rejecting it.

Reply:
Hmm, fair point. If it's not crazy to capture that auth in the code, go for it. If it is crazy, then we can ignore it.

Reply:
It isn't too hard to do this check (yet another payload validation), but I think we can rely on the app here for simplicity.

Reply:
Whoops, I may have been confusing. I meant if it's hard to tackle the edge case you mentioned, we can ignore the edge case. I think we should authorize the teleconsultation_id here regardless. What do you think

Reply:
Yeah, the authorisation can't be done basically without losing on the edge case information. I'd say we let this field be liberal.